client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -836,6 +821,7 @@ const mapStateToProps = (
 		...anchorFmData,
 		openSidebar: getQueryArg( window.location.href, 'openSidebar' ),
 		showDraftPostModal,
+		...pressThis,
```

Comment:
[NIT] Prefer to rename to `pressThisData` as `pressThis` looks like a function and doing spreading is a bit weird 😅

Reply:
good point